projects/angular/src/data/datagrid/utils/key-navigation-grid.controller.ts

```diff
@@ -186,10 +193,11 @@ export class KeyNavigationGridController implements OnDestroy {
     activeCell.setAttribute('tabindex', '0');
     this._activeCell = activeCell;
 
-    const items = getTabableItems(activeCell);
-    const item = activeCell.getAttribute('role') !== 'columnheader' && items[0] ? items[0] : activeCell;
+    const actionableItems = getTabableItems(activeCell);
+    const item =
+      activeCell.getAttribute('role') !== 'columnheader' && actionableItems[0] ? actionableItems[0] : activeCell;
 
-    if (!this.skipItemFocus) {
+    if (!(this.skipItemFocus || keepFocus)) {
```

Comment:
```suggestion
    if (!this.skipItemFocus && !keepFocus)) {
```

I think it's more readable this way.